curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da3c6b6fbd742bff6ee40.md

```diff
@@ -7,7 +7,11 @@ dashedName: step-2
 
 # --description--
 
-The <dfn>Fetch API</dfn> is a built-in JavaScript interface to make network requests to a server. It has a `fetch()` method you can use to make `GET`, `POST`, `PUT`, or `PATCH` requests. In this project, you'll make a `GET` request to a URL for a JSON file with information about authors on freeCodeCamp News.
+In this project we want data about
+the authors on freeCodeCamp News. If you want data from an online source, you need use an `API` (Application Programming Interface). An `API` lets
+people from outside of an organization retrieve its internal data. 
```

Comment:
```suggestion
In this project we want data about the authors on freeCodeCamp News.
If you want data from an online source, you need use an `API` (Application Programming Interface).
An `API` lets people from outside of an organization retrieve its internal data. 
```

it doesn't change anything in the challenge, but breaking the sentence like that was bothering me